src/main.js

```diff
@@ -0,0 +1,12 @@
+import FiltersView from './view/filters-view.js';
+import {render} from './render.js';
+import TripBoardPresenter from './presenter/trip-board-presenter.js';
+
+const filtersContainerElement = document.querySelector('.trip-controls__filters');
```

Comment:
Мы не использует тип на названии переменных
```suggestion
const filtersContainer = document.querySelector('.trip-controls__filters');
```

Reply:
https://up.htmlacademy.ru/ecmascript/19/criteries#b9